### PR TITLE
[UI] add help tooltip to icon

### PR DESCRIPTION
**Pull request by bariqhibat**
Signed-off-by: Bariq <bariqhibat@gmail.com>

**Description**

**Notes for Reviewers**
Some are lines are because of linting issue.
Meshery-extension 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
@bariqhibat if you could add an ss or a gif of the UI change, it will be helpful to review. Code wise things look fine to me.

File: ui/components/MesheryMeshInterface/PatternServiceForm.js [hunk 1]
```diff
@@ -148,6 +150,15 @@ function PatternServiceForm({ formData, schemaSet,onChange, onSubmit, onDelete,
         <AppBar>
           <Toolbar variant="dense" style={{ padding : "0 10px" }}>
             <p style={{ margin : "auto auto auto 10px", fontSize : "18px" }}>{schemaSet.workload.title}</p>
+            {schemaSet?.workload?.description && (
+              <label htmlFor="help-button">
```
Comment (Nikhil-Ladha) [inline ui/components/MesheryMeshInterface/PatternServiceForm.js:154]: Does this tag really play any role?

Comment (bariqhibat) [reply ui/components/MesheryMeshInterface/PatternServiceForm.js:154]: Not really sure, I copied this from the codebase

**Comment by bariqhibat** [comment]
> @bariqhibat if you could add an ss or a gif of the UI change, it will be helpful to review. Code wise things look fine to me.

image

File: ui/components/MesheryMeshInterface/PatternServiceForm.js (hunk 1 above)
Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternServiceForm.js:154]: Beats me!

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you, @bariqhibat !

**Pull request merged**